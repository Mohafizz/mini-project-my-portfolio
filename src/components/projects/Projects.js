import React, { Component } from "react";
import "./Projects.css";
import TodoListCard from "../project-cards/TodoListCard";
import ProjectCard from "../project-cards/ProjectCard";
import { Grid, Cell } from "react-mdl";
import UpcomingProjectCard from "../project-cards/UpcomingProjectCard";
class Projects extends Component {
  render() {
    return (
      <div>
        <div style={{ width: "100%", margin: "auto" }}>
          <Grid className="projects-grid">
            <Cell col={4} tablet={6}>
              <ProjectCard
                imagePath="html-css-blog.jpg"
                projectUrl="https://mohafizz.github.io/html_css/"
              />
            </Cell>
            <Cell col={4} tablet={6}>
              <TodoListCard />
            </Cell>
            <Cell col={4} tablet={6}>
              <UpcomingProjectCard />
            </Cell>
          </Grid>
        </div>
      </div>
    );
  }
}

export default Projects;
