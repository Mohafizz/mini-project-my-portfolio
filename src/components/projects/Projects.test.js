import React from "react";
import { shallow } from "enzyme";
import Projects from "./Projects";

const project = shallow(<Projects />);

it("should render correctly", () => {
  expect(project).toMatchSnapshot();
});
