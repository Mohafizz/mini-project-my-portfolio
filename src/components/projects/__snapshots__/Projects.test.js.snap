// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Projects />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <div
        style={
                Object {
                        "margin": "auto",
                        "width": "100%",
                      }
        }
>
        <Grid
                className="projects-grid"
        >
                <Cell
                        col={4}
                        tablet={6}
                >
                        <HtmlCssBlogCard />
                </Cell>
                <Cell
                        col={4}
                        tablet={6}
                >
                        <TodoListCard />
                </Cell>
                <Cell
                        col={4}
                        tablet={6}
                >
                        <UpcomingProjectCard />
                </Cell>
        </Grid>
</div>,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <Grid
          className="projects-grid"
>
          <Cell
                    col={4}
                    tablet={6}
          >
                    <HtmlCssBlogCard />
          </Cell>
          <Cell
                    col={4}
                    tablet={6}
          >
                    <TodoListCard />
          </Cell>
          <Cell
                    col={4}
                    tablet={6}
          >
                    <UpcomingProjectCard />
          </Cell>
</Grid>,
        "style": Object {
          "margin": "auto",
          "width": "100%",
        },
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "children": Array [
            <Cell
              col={4}
              tablet={6}
>
              <HtmlCssBlogCard />
</Cell>,
            <Cell
              col={4}
              tablet={6}
>
              <TodoListCard />
</Cell>,
            <Cell
              col={4}
              tablet={6}
>
              <UpcomingProjectCard />
</Cell>,
          ],
          "className": "projects-grid",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "children": <HtmlCssBlogCard />,
              "col": 4,
              "tablet": 6,
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {},
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "children": <TodoListCard />,
              "col": 4,
              "tablet": 6,
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {},
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "children": <UpcomingProjectCard />,
              "col": 4,
              "tablet": 6,
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {},
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            "type": [Function],
          },
        ],
        "type": [Function],
      },
      "type": "div",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          style={
                    Object {
                              "margin": "auto",
                              "width": "100%",
                            }
          }
>
          <Grid
                    className="projects-grid"
          >
                    <Cell
                              col={4}
                              tablet={6}
                    >
                              <HtmlCssBlogCard />
                    </Cell>
                    <Cell
                              col={4}
                              tablet={6}
                    >
                              <TodoListCard />
                    </Cell>
                    <Cell
                              col={4}
                              tablet={6}
                    >
                              <UpcomingProjectCard />
                    </Cell>
          </Grid>
</div>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <Grid
            className="projects-grid"
>
            <Cell
                        col={4}
                        tablet={6}
            >
                        <HtmlCssBlogCard />
            </Cell>
            <Cell
                        col={4}
                        tablet={6}
            >
                        <TodoListCard />
            </Cell>
            <Cell
                        col={4}
                        tablet={6}
            >
                        <UpcomingProjectCard />
            </Cell>
</Grid>,
          "style": Object {
            "margin": "auto",
            "width": "100%",
          },
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "children": Array [
              <Cell
                col={4}
                tablet={6}
>
                <HtmlCssBlogCard />
</Cell>,
              <Cell
                col={4}
                tablet={6}
>
                <TodoListCard />
</Cell>,
              <Cell
                col={4}
                tablet={6}
>
                <UpcomingProjectCard />
</Cell>,
            ],
            "className": "projects-grid",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "children": <HtmlCssBlogCard />,
                "col": 4,
                "tablet": 6,
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "function",
                "props": Object {},
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "children": <TodoListCard />,
                "col": 4,
                "tablet": 6,
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "function",
                "props": Object {},
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "children": <UpcomingProjectCard />,
                "col": 4,
                "tablet": 6,
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "function",
                "props": Object {},
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
              "type": [Function],
            },
          ],
          "type": [Function],
        },
        "type": "div",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
